src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
+@import "reset.scss";
+
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+$dark-color: #060b35;
+$grey-color: #616070;
+$white-color: #fff;
+$blue-color: #00acdc;
```

Comment:
you need to fix the variable names, they should show their purpose, not their content. if I decided to change the color, I would have to change it everywhere, not just in the variable definition.

you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)